.appveyor.yml

```
@@ -0,0 +1,53 @@
+# general configuration
+version: '{branch}.{build}'
+
+# environment configuration
+image: Visual Studio 2017
+clone_folder: C:\projects\php_simple_kafka_client
+environment:
+  BIN_SDK_VER: 2.2.0
+  DEP: librdkafka-1.5.3
```

```suggestion
  DEP: librdkafka-1.6.1
```
Iif it is available, it would be great to use the latest version
I setup the project on AppVeyor, i hope the next change triggers the build, manually i was somehow only able to build the main branch :sweat_smile:

---

Unfortunately, 1.5.3 is the newest version available for now. Since these libs have to be built manually, I'm hoping to switch to [vcpkg](https://github.com/microsoft/vcpkg), but they have only 1.5.0, so I suggest to stick with 1.5.3 for a while.

> manually i was somehow only able to build the main branch

Yes, that's a bit contrieved. You might need to change the `default branch` under `Settings` to be able to manually trigger a build. However, there is a [build in my fork](https://ci.appveyor.com/project/cmb69/php-simple-kafka-client/builds/38660085) which succeeded (basically this PR + PR #40). Anyhow, I'm going to push a rebased version of this patch right now.

---

AppVeyor CI is running. \o/